### ISSUE TITLE
Repeat mode

### ISSUE THREAD

**Issue by Kry256** [spotify/ios-sdk#795]
Hi, I just figured out that in SDK beta 24 there is the possibility to set 3 different "repeat modes"

`SPTRepeatOff`
`SPTRepeatContext`
`SPTRepeatOne`

But delegate and `SPTAudioStreamingController` only tell me if repeat mode is active or not with a boolean value.
Is there any way to retrieve the value previously setted up with

`- (void)setRepeat:(SPTRepeatMode)mode callback:(SPTErrorableOperationCallback)block` ?

**Comment by erdtman** [comment]
There is a difference between repeat enabled (on/off) and repeat mode (SPTRepeatOff/SPTRepeatContext/SPTRepeatOne) to get the latest repeat mode you can subscribe to didChangeRepeatStatus

**Comment by Kry256** [comment]
Yeah, I saw the delegate method `didChangeRepeatStatus` (sorry didn't noticed it before).
But, outside that method, how can I retrieve the actual setted `repeatMode` value?e?
